<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newSales</class>
 <widget class="QWidget" name="newSales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>649</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>649</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>462</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>462</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>24</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Select</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Day 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Day 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Day 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Day 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Day 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Day 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Day 7</string>
    </property>
   </item>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>43</y>
     <width>591</width>
     <height>411</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>591</width>
     <height>411</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>591</width>
     <height>411</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>589</width>
      <height>409</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>411</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>591</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>591</width>
       <height>411</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
